.stylelintrc.yaml

```diff
@@ -18,6 +18,16 @@ rules:
   # Double-quotes are our convention throughout our codebase within (S)CSS. They also reflect how
   # attribute strings are normally quoted within the DOM.
   string-quotes: double
+  order/order:
+    -
```

Comment:
Sorry if I'm being ignorant of some feature of yaml, but what happened with these dashes on their own lines here?

Reply:
the dashes create lists, and non dash is like a key in a map. Its basically JSON, but weird (I think).

And I need this format in order for the stylelint-order plugin to parse it